lighthouse-core/gather/gatherers/script-elements.js

```diff
@@ -92,47 +64,28 @@ class ScriptElements extends FRGatherer {
       ],
     });
 
-    for (const script of scripts) {
-      if (mainResource && script.content) script.requestId = mainResource.requestId;
-    }
-
     const scriptRecords = networkRecords
+      .filter(record => record.resourceType === NetworkRequest.TYPES.Script)
       // Ignore records from OOPIFs
-      .filter(record => !record.sessionId)
```

Comment:
should we continue to ignore records from OOPIFs? this would only impact pubads.

I'd think pubads would want any and all scripts so it makes sense to remove this filter?

Reply:
We should probably ask them. I think it makes sense to leave as is for this PR